src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+function salaryToNumber(salary) {
+  return +salary
+    .split('')
+    .filter((char) => '1234567890'.includes(char))
+    .join('');
+}
+
+function sortList(list) {
+  const sortEmployees = list;
```

Comment:
The variable `sortEmployees` is unnecessary since it directly references the input `list`. You can call `list.sort(...)` directly without assigning it to another variable.